torch/nn/modules/padding.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, value: float) -> None:
         self.value = value
 
     def forward(self, input: Tensor) -> Tensor:
-        return F.pad(input, self.padding, 'constant', self.value)
+        return F.pad(input, self.padding, 'constant', self.value)  # type: ignore[arg-type]
```

Comment:
Why is the `ignore` necessary here?

Reply:
mypy infers the type of `self.padding` as `Union[Tensor, Module]` but `F.pad` expects padding as a `Sequence[int]`.

```
torch/nn/modules/padding.py:21: error: Argument 2 to "pad" has incompatible type "Union[Tensor, Module]"; expected "Sequence[int]"  [arg-type]
```

Replacing `# type: ignore[...]` by an assertion also crashes pytorch. I think I will defer annotating this module for now since it requires more work